job-runner/main.go

```diff
@@ -14,9 +14,21 @@ import (
 )
 
 func main() {
+	// Make a chain repo.
+	chainRepo := chain.NewMemoryRepo()
```

Comment:
At some point you will be able to set all of this easily in a config file, but we don't have such a thing in place yet so if you want to change your repo type away from the memory repo right now you have to edit main.go (see comments below this line)